### DISCUSSION TITLE
Why can't zotero better-bibtex generate better-bibtex-search. sqlite in zotero beta 7.0

### DISCUSSION THREAD

**Discussion by chenchenxl** [retorquere/zotero-better-bibtex#2686] [General]
I am using the zotro zotlit plugin and it requires a **better bibtex search. sqlite** file, but better bibtex cannot generate this file. I have two computers with the same Zotero and plugin versions, but one of them can generate this file

**Comment by retorquere** [comment]
Please open a issue and I'll have a look.

**Reply by chenchenxl** [reply]
Please take a look at this link:https://github.com/retorquere/zotero-better-bibtex/issues/2687

**Reply by retorquere** [reply]
Sorry, I didn't read your question properly. Recent BBTs no longer generate a separate search database, the main database has the citekeys now.

**Reply by retorquere** [reply]
See also https://github.com/PKM-er/obsidian-zotlit/issues/236. The zotlit plugin will have to be updated.
